crates/fluvio-controlplane/src/sc_api/update_lrs.rs

```diff
@@ -62,6 +62,7 @@ pub struct LrsRequest {
     pub leader: ReplicaStatus,
     pub replicas: Vec<ReplicaStatus>,
     pub size: i64,
+    pub base_offset: i64,
```

Comment:
Try something like this
```suggestion
   #[fluvio(min_version = 1)]
    pub base_offset: i64,
```

and in the trait impl where we have:

```
impl Request for UpdateLrsRequest {

}
```
Add this: 
```
const DEFAULT_API_VERSION: i16 = 1;
```

Reply:
Done!

Reply:
Given that we used `#[fluvio(min_version = 15)]` on `PartitionStatus` https://github.com/infinyon/fluvio/pull/4059/files#diff-f1e14778be799260d11d5529f1a8c3a5796cf26e4a8993eab058408b7b828a08L26, I think we should use the same value `15` for `pub base_offset: i64,` here?

Reply:
This way we also match against the new min version:
https://github.com/infinyon/fluvio/pull/4059/files#diff-b7ec42b71799d984a63822863daca2ce587a551f59133cafd64410e667df49c4R16

Reply:
Did that but it still seems that the upgrade is not correct.
```
Validate test data w/ v0.11.10-dev-1 CLI matches expected data AFTER upgrading cluster + CLI to v0.11.10-dev-1
shasum: WARNING: 1 computed checksum did NOT match
-: FAILED
Got: 
Expected: 22e48620de3e901be2d32b33c5f4430a67fc64ba
```

Can it be the global version?